### PR TITLE
[ID] Rework and Update FAQ

### PR DESCRIPTION
**Pull request by Nivalyx**
continuation of [#4166](http://github.com/ppy/osu-wiki/pull/4166) and [#4229](http://github.com/ppy/osu-wiki/pull/4229).

i'm aware that the original FAQ article is in a need of a cleanup somewhat but since the ID FAQ page as it stands is sooooooooooo outdated (and is missing a huge lot of critical information) i decided to go for it anyway 😭

as before i've double-checked everything and this pull request should hopefully be good to be merged through and through but reviews are welcomed as always.

### CODE REVIEW

File: wiki/FAQ/id.md [hunk 1]
```diff
@@ -1,204 +1,508 @@
...
+
+Posisi kursormu tidak memiliki pengaruh apapun dalam permainan osu!catch kecuali apabila kamu bermain dengan menggunakan mod [Relax](/wiki/Game_modifier/Relax). Pada mod ini, alih-alih mengikuti input dari keyboard posisi catcher milikmu akan berpindah seiring dengan pergerakan kursor mouse yang kamu buat.
+
+### Apa itu osu!mania?
+
+*Penjelasan lebih lanjut: [osu!mania](/wiki/Game_mode/osu!mania)*
+
+![Tampilan antarmuka permainan osu!mania](/wiki/shared/mania-gameplay.jpg "Tampilan antarmuka permainan osu!mania")
+
+osu!catch merupakan mode permainan osu! yang didasari pada permainan-permainan *vertical scrolling rhythm game* (VSRG) lainnya seperti *beatmania*, *Guitar Hero*, dan *Dance Dance Revoulution*. Tidak jauh berbeda dengan permainan-permainan tersebut, mode permainan osu!mania menuntut kamu untuk dapat menekan tombol input yang sesuai pada keyboard kamu pada waktu yang tepat sejalan dengan bergulirnya not-not yang ada pada layar permainan.
```
Comment (troke12) [inline wiki/FAQ/id.md:218]: osu!catch atau osu!mania ? 😅
```suggestion
osu!mania merupakan mode permainan osu! yang didasari pada permainan-permainan *vertical scrolling rhythm game* (VSRG) lainnya seperti *beatmania*, *Guitar Hero*, dan *Dance Dance Revoulution*. Tidak jauh berbeda dengan permainan-permainan tersebut, mode permainan osu!mania menuntut kamu untuk dapat menekan tombol input yang sesuai pada keyboard kamu pada waktu yang tepat sejalan dengan bergulirnya not-not yang ada pada layar permainan.
```

File: wiki/FAQ/id.md [hunk 2]
```diff
@@ -1,204 +1,508 @@
...
+
+#### Penjelasan dasar
+
+##### Layar permainan
+
+![Tampilan layar permainan osu!mania](/wiki/shared/Mania_playfield.jpg "Tampilan layar permainan osu!mania")
+
+Secara *default*, not-not yang ada dalam permainan osu!mania akan secara perlahan bergulir dari atas ke bawah layar. Sebelum memulai permainan, kamu akan dihadapkan dengan sebuah anak panah yang akan menunjukkan arah perguliran not-not yang akan kamu hadapi. Kamu dapat mengubah arah perguliran not-not tersebut menjadi dari bawah ke atas melalui menu `Options`; cari dan pilih `osu!mania layout` dari daftar pilihan yang tersedia lalu centang `Vertically flip playfield (DDR style)` pada layar yang muncul.
```
Comment (troke12) [inline wiki/FAQ/id.md:228]:
```suggestion
Secara *default*, not-not yang ada dalam permainan osu!mania akan secara perlahan turun dari atas ke bawah layar. Sebelum memulai permainan, kamu akan dihadapkan dengan sebuah anak panah yang akan menunjukkan arah perguliran not-not yang akan kamu hadapi. Kamu dapat mengubah arah perguliran not-not tersebut menjadi dari bawah ke atas melalui menu `Options`; cari dan pilih `osu!mania layout` dari daftar pilihan yang tersedia lalu centang `Vertically flip playfield (DDR style)` pada layar yang muncul.
```

File: wiki/Welcome/id.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Selamat Datang
 
-Selamat datang di osu!, permainan ritme free-to-win terbaik yang dikembangkan oleh peppy dengan empat mode permainan yang dapat kamu nikmati di dalamnya: osu!standard, bagi para pemain yang ingin mengeklik lingkaran; osu!taiko, bagi para pemain yang ingin bermain drum; osu!catch, bagi para pemain yang ingin menangkap buah; dan osu!mania, bagi para pemain yang ingin menekan tuts-tuts keyboard mengikuti irama lagu.
+Selamat datang di osu!, game ritme free-to-win terbaik yang dikembangkan oleh peppy dengan empat mode permainan yang dapat kamu nikmati di dalamnya: osu!standard, bagi para pemain yang ingin bermain dengan lingkaran; osu!taiko, bagi para pemain yang ingin menabuh drum; osu!catch, bagi para pemain yang ingin menangkap buah; dan osu!mania, bagi para pemain yang ingin menekan tuts-tuts keyboard mengikuti irama lagu.
```
Comment (troke12) [inline wiki/Welcome/id.md:3]: bukannya gim?
```suggestion
Selamat datang di osu!, gim ritme free-to-win terbaik yang dikembangkan oleh peppy dengan empat mode permainan yang dapat kamu nikmati di dalamnya: osu!standard, bagi para pemain yang ingin bermain dengan lingkaran; osu!taiko, bagi para pemain yang ingin menabuh drum; osu!catch, bagi para pemain yang ingin menangkap buah; dan osu!mania, bagi para pemain yang ingin menekan tuts-tuts keyboard mengikuti irama lagu.
```

Comment (Nivalyx) [reply wiki/Welcome/id.md:3]: ah, actually i changed `permainan` into `game` for this line in order to make it consistent with what has been written in the main page which is `game ritme free-to-win terbaik` o.o

image

**Comment by Nivalyx** [comment]
applied @troke12's suggestions!

**Review by TPGPL — CHANGES_REQUESTED** [review]
- checked from the first few lines and the translation is missing content from the original (maybe even 30 lines)
- image alt texts are equal to the tooltips, which is different from the original

**Comment by Nivalyx** [comment]
fixed the image alt texts

i can guarantee that no information is missing from the original document tho... the "missing lines" if any is caused by some of the lines being merged into a single paragraph for better context and clarity in the destination language (Indonesian) ._.

File: wiki/FAQ/id.md [hunk 4]
```diff
@@ -1,204 +1,508 @@
...
+
+Secara *default*, input-input yang terdaftar pada osu! adalah sebagai berikut: 
+
+| Mouse | Keyboard | Tablet/Touchscreen |
+| :-- | :-- | :-- |
+| Klik kiri(M1) / Klik kanan(M2) | `Z`(K1) / `X`(K2) | Sentuhan pada layar(M1) |
+
+Pada umumnya, osu! dapat menerima input dalam bentuk apapun dari peranti apapun selama kamu memberikan input yang bersangkutan pada waktu dan lokasi yang tepat.
+
+Apabila kamu bermain dengan menggunakan mod [Relax](/wiki/Game_modifier/Relax), kamu hanya perlu untuk menggerakkan kursormu tanpa harus memberikan input apapun ke dalam osu!. Sebaliknya, apabila kamu bermain dengan menggunakan mod [AutoPilot](/wiki/Game_modifier/Autopilot), kamu hanya perlu untuk memberikan input ke dalam osu! tanpa perlu menggerakkan kursormu.
```
Comment (TPGPL) [inline wiki/FAQ/id.md:79]: Spun Out link missing
original:
```md
If [Relax](/wiki/Game_modifier/Relax) game modifer was used, only the in-game cursor will work. Use the in-game cursor to follow the hit objects with automated tap. Spinners must still be completed.

If [Auto Pilot](/wiki/Game_modifier/Autopilot) game modifier was used, only the input from the input device will work. Time the tap on the hit objects with automated cursor movement. Spinners will follow the [Spun Out](/wiki/Game_modifier/Spun_Out) mod speed.```

File: wiki/FAQ/id.md [hunk 5]
```diff
@@ -1,204 +1,508 @@
...
+Performance point(pp) merupakan suatu nilai yang digunakan untuk mengukur kemampuan seorang pemain relatif terhadap rekam jejak permainan dari pemain yang bersangkutan. Masing-masing mode permainan memiliki mekanisme perhitungan pp-nya tersendiri, namun pada dasarnya **semakin sulit suatu beatmap, semakin besar pula pp yang dapat kamu raih dari beatmap tersebut**. Untuk dapat memperoleh pp, kamu harus senantiasa menantang dirimu untuk dapat memainkan dan menguasai beatmap-beatmap dengan tingkat kesulitan yang terus meningkat seiring waktu. Harap diingat bahwa nilai pp yang kamu miliki berlaku secara terpisah per mode permainan; pp yang kamu dapatkan dari bermain beatmap osu!catch hanya akan berpengaruh pada nilai pp osu!catch kamu dan lain sebagainya.
+
+Untuk informasi yang lebih mendalam seputar pp, harap kunjungi laman [Performance Point](/wiki/Performance_Points).
+
+---
+
+## Pertanyaan-Pertanyaan Lainnya yang Umum Ditanyakan
+
+### Topik-topik umum
+
```
Comment (TPGPL) [inline wiki/FAQ/id.md:365]: there was a new question added recently
```md
#### I think I've found a bug! Where can I report it?

If you have a general technical issue, or if you're unsure whether the issue you've encountered is a bug or not, you can create a post in the [Help forum](https://osu.ppy.sh/community/forums/5). If you think you've found a specific, critical issue with osu!, you can create an issue in the [osu-stable-issues GitHub repository](https://github.com/ppy/osu-stable-issues/issues). Before posting, make sure that nobody has already reported your problem.
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Comment by Nivalyx** [comment]
i wasn't aware about the recently added line in the FAQ about troubleshooting or the missing SpunOut reference oops

added both 👍

**Review by TPGPL — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**